reviewrot/pagurestack.py

```diff
@@ -155,16 +155,40 @@ def get_last_comment(self, res):
                 body=str(comments[-1]['comment']),
                 created_at=last_comment_date)
 
-    @staticmethod
-    def _avatar(username):
-        """ Return the avatar of a given pagure user.
+    def _avatar(self, username, ssl_verify=True):
+        """ Return the avatar URL of a given pagure user.
```

Comment:
Negligible, but since we are changing this, can we please also add the following to doc string to be consistent across the app?
```
 Args: 
      username: 
```

Reply:
Done!